homeassistant/components/frontend/__init__.py

```diff
@@ -569,7 +569,9 @@ def get_template(self) -> jinja2.Template:
         """Get template."""
         tpl = self._template_cache
         if tpl is None:
-            with open(str(_frontend_root(self.repo_path) / "index.html")) as file:
+            with open(
+                str(_frontend_root(self.repo_path) / "index.html"), encoding="utf8"
+            ) as file:
```

Comment:
Guess not for this PR, but this can be rewritten as
```suggestion
            with (_frontend_root(self.repo_path) / "index.html").open(encoding="utf8") as file:
```

Reply:
👍🏻 I wasn't aware  that  it's a `pathlib.Path` object. Pushed a commit to update it